Packs/BitwardenPasswordManager/Integrations/BitwardenPasswordManagerEventCollector/BitwardenPasswordManagerEventCollector.py

```diff
@@ -0,0 +1,280 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from CommonServerUserPython import *  # noqa
+
+VENDOR = 'Bitwarden'
+PRODUCT = 'Password Manager'
+
+DEFAULT_MAX_FETCH = 500
+DEFAULT_FIRST_FETCH = '3 days'
+MINUTES_BEFORE_TOKEN_EXPIRED = 2
+
+AUTHENTICATION_FULL_URL = 'https://identity.bitwarden.com/connect/token'
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+class Client(BaseClient):
+    """
+    Client class to interact with the service API
+    """
+
+    def __init__(self, base_url: str, verify: bool, proxy: bool, client_id: str, client_secret: str):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy)
+        self.token = self.login(client_id, client_secret)
+
+    def login(self, client_id: str, client_secret: str) -> str:
+        integration_context = get_integration_context()
+        demisto.log(f"{integration_context=}")
+        if token := integration_context.get('token'):
+            expires_date = integration_context.get('expires')
+            if expires_date and not self.is_token_expired(expires_date):
+                demisto.log(f"Token is valid: {expires_date=}, {token=}")
```

Comment:
```suggestion
                demisto.debug(f"Token is valid: {expires_date=}")
```
we don't write sensitive data in logs